### ISSUE TITLE
Player resizing causing two audio streams.

### ISSUE THREAD

**Issue by maiofalmo** [YePpHa/YouTubeCenter#1845]
YTC Dev Build 525 - Firefox addon
Firefox 38.0.1 

When loading long videos (1 hour+) there is two audio streams created with one lagging slightly behind the other. It occurs when the video player resizes.

Disabling "Enable resize" in the "Player Size" tab fixes the problem upon refreshing the page. After turning resizing back on, the problem is back.

After disabling "Enable resize" this introduces more than 3 seconds of lag as the video player resizes anyway. The dupe audio also comes back intermittently but not every single time like with resizing enable.

**Comment by DannyPockets** [comment]
Does the second audio continue playing even after you pause the video?

**Comment by maiofalmo** [comment]
Yes it does. I eventually gave up on all the issues and disabled until the issues are fix.ed

**Comment by ireun** [comment]
Haven't you forced a flash player? I've seen an issue that it was this that caused this, if so try using HTML5 player..
